app/views/housing_reviews/_form.html.erb

```diff
@@ -51,6 +51,13 @@
     </div>
   </div>
 
+  <div class="field">
+  <%= form.label "images", "Images of the room:" %>
+    <div class="control">
+    <%= form.file_field :images, multiple: true, accept: 'image/jpeg,image/png,image/gif,application/pdf', :placeholder => "Upload your file" %>
```

Comment:
let's remove gifs and pdfs. though the gifs would render and pdfs might not, I don't think they would be needed in the housing reviews